### ISSUE TITLE
Align Bedingung / Hinweis / Format entries

### ISSUE THREAD

**Issue by hf-krechan** [Hochfrequenz/ahb-tabellen#129]
The entries in the column `Bedingung / Hinweis / Format` should not like

```
[902] Format: Möglicher Wert: ≥ 0 [937] Format: keine Nachkommastelle
```
instead the user would like to see
```
[902] Format: Möglicher Wert: ≥ 0
[937] Format: keine Nachkommastelle
```

example: https://ahb-tabellen.stage.hochfrequenz.de/ahb/FV2410/55001

**Comment by hf-krechan** [comment]
It could also be an issue for kohlrahbi. In most cases it looks already fine.

**Comment by OLILHR** [comment]
fixed by #235
